.github/actions/scale-tests/validate-cluster/action.yaml

```diff
@@ -0,0 +1,20 @@
+name: 'Validate cluster'
+description: "Workflow that wait for cluster to be in a ready state"
+inputs:
+  cluster_name:
+    description: "Name of the cluster"
+    required: true
+  kops_state:
+    description: "Bucket with kops state"
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - name: Validate cluster
+      shell: bash
+      run: |
+        ./kops validate cluster \
+          ${{ inputs.cluster_name }} \
+          --state ${{ inputs.kops_state }} \
+          --wait 20m \
```

Comment:
Nit: this timeout value is hard coded, but a user could specify a longer timeout when consuming the workflow through the GHA option `timeout-minutes`.